neptune/alpha/internal/containers/disk_queue.py

```diff
@@ -61,32 +70,45 @@ def put(self, obj: T) -> None:
         if self._file_size + len(_json) > self._max_file_size:
             self._writer.close()
             self._write_file_idx += 1
-            self._writer = open(self._current_write_log_file(), "a")
+            self._writer = open(self._get_log_file(self._write_file_idx), "a")
```

Comment:
Not really sure about this one, but is the cost of opening a file for each operation negligible?

Reply:
Not for each operation. It opens next log file if previous one is too big.